apps/design/frontend/src/layout.tsx

```diff
@@ -31,16 +31,28 @@ export const FieldName = styled.div`
   margin-bottom: 0.5rem;
 `;
 
+export const FieldTag = styled.span`
+  color: ${(p) => p.theme.colors.neutral};
+  font-size: 0.8rem;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.regular};
+  margin-bottom: 0.5rem;
+`;
+
 export function InputGroup({
   label,
   children,
+  isOptional = false,
 }: {
   label: string;
   children: React.ReactNode;
+  isOptional?: boolean;
 }): JSX.Element {
   return (
     <label>
-      <FieldName>{label}</FieldName>
+      <div style={{ display: 'flex', alignItems: 'center', gap: '0.25rem' }}>
+        <FieldName>{label}</FieldName>
+        {isOptional && <FieldTag> — Optional</FieldTag>}
```

Comment:
Will post in Slack for convo on the right ui design for these as optional fields